Documentation/ceph-monitoring.md

```diff
@@ -91,6 +95,7 @@ _Note: This expects prometheus to be pre-installed by the admin._
 
 To enable prometheus alerts after cluster is deployed, run the following commands:  
 ```BASH
+kubectl create -f monitoring-roles.yaml
 cd cluster/examples/kubernetes/ceph/monitoring
```

Comment:
Does the Ceph cluster resource not need modified to enable monitoring after deployment? e.g., `kubectl apply -f cluster.yaml` after making the above modifications

Reply:
Updated doc to remove this confusion. Kept it generic enough for any k8s admin to understand.